apps/communications/dialer/test/unit/dialer_test.js

```diff
@@ -826,44 +829,4 @@ suite('navigation bar', function() {
       sinon.assert.called(NavbarManager.show);
     });
   });
-
```

Comment:
We shouldn't be removing this suite, just the check that we load the needed resources before using them.